src/app-layout/visual-refresh/trigger-button.scss

```diff
@@ -21,9 +21,8 @@ handleSplitPanelMaxWidth function in the context.
   cursor: pointer;
   height: awsui.$space-layout-toggle-diameter;
   padding: 0 awsui.$space-s;
-  width: awsui.$space-layout-toggle-diameter;
-
   pointer-events: auto;
+  width: awsui.$space-layout-toggle-diameter;
```

Comment:
Out of curiosity, what tool are you using the reorder CSS properties? Should we make it a default in our style-lint config?

Reply:
I am using this amazing and fancy tool called "Scott's neurotic hands", and yes we should absolutely update our style-lint config to alphabetize CSS properties!

Reply:
I'm not sure how I feel about alphabetical. But there are other more "elaborate" conventions like idiomatic-css: https://github.com/necolas/idiomatic-css#declaration-order.

Reply:
To each his own! It likely depends on the problem the standard is attempting to solve. Personally, my goal is to scan a list and find the presence (or lack thereof) of one or more properties as quickly as possible. For me alphabetical ordering has consistently won out compared to logical ordering. If the goal were to derive meaning (i.e. "does this set of properties impact layout?") logical order could be more effective.